components/banners/WhatsNewBanner.tsx

```diff
@@ -13,7 +13,7 @@ export function WhatsNewBanner({
   return (
     <Alert variant="primary" style={{ textAlign: "center", margin: 0 }}>
       <small>
-        <strong>What's New</strong> - {releaseNote.description}{" "}
+        <strong>What&apos;s New</strong> - {releaseNote.description}{" "}
```

Comment:
I'd probably go with quoted strings here `{"what's new"}` vs html codes

Reply:
Personally, I find this kind of annoying and would rather turn this rule off - TS compiling helps know if our strings are terminated or not

Reply:
Happy to turn it off!